ai_genomics/analysis/data_scoping/patent_data/search_evaluation.md

```diff
@@ -0,0 +1,64 @@
+
+The table below shows the counts of projects in the sample that are and are not related to AI and genomics. It is immediately clear that there are many projects that are not related to the two subjects - 66 are related to only one of them, and 7 are related to neither. The content of the titles, abstracts and patent texts suggests some reasons as to why this may be.
+
+|        | Not Genomics | Genomics |
+| :----- | -----------: | -------: |
+| Not AI |            7 |       41 |
+| AI     |           25 |       27 |
+
+One reason is that the definitions of AI and genomics are overly broad. The current filtering mechanism includes patents predicted to relate to machine learning, natural language processing, computer vision, speech, knowledge processing, AI hardware, evolutionary computation, and planning and control. Several patents covered topics such as new silicon chip architectures and automated manufacturing processes, which are more tangentially related to enabling and employing AI. In addition, there were several patents that covered genetic algorithms, but did not touch upon genomics. Upon closer inspection, the patent codes for these documents often referred to 'genetic computing'.
+
+Another possible reason is poor classification of AI topics in the USPTO data. Several examples in the dataset discussed topics that contained vocabulary associated with AI and machine learning but that were being used in other contexts. For example, one patent referred to 'clustering' several times but was in fact referring to the physial aggregation of biological particles. In several cases, the word 'data' is used throughout, but the methods described to use the data are not based on machine learning. There were also some patents that used conventional computer vision, reliant on conventional image processing rather than AI-like technologies. It is not immediately clear which part of the pipeline used to predict AI patents that these errors originate from.
+
+In addition to counting the number of patents that related to our two areas of interest, the number of patents that had ambiguous tites or abstracts were recorded. We can see that in the minority of cases where Only 27 that are both AI and genomics. 66 are only one and 7 are neither. This suggests that we should be careful when performing text analysis (e.g. topic modelling) on the patent abstracts as they do not reveal all of the information about the patent that might be most relevant for our purpose.
```

Comment:
agreed on relying only on abstracts - when @Jack-0-0 and I labelled ai and genomics patents at different thresholds, we found we were often referring to patents.google.com for more context beyond abstracts